notebook/static/notebook/js/mathjaxutils.js

```diff
@@ -178,6 +178,11 @@ define([
                     end = block;
                     braces = 0;
                 }
+                else if (block === "\\\\\(") {
```

Comment:
This modification should also cover the case where `block === "\\\\\["` and `end = "\\\\\]"`.